### ISSUE TITLE
[Bug] Unknown Cmake Options in wiki

### ISSUE THREAD

**Issue by dnns92** [openvinotoolkit/openvino#9773]
##### System information (version)

- OpenVINO=> 2021.4
- Operating System / Platform => Win10
- Compiler => doesnt matter
- Problem classification => Build-Problem

##### Detailed description
There are options here: https://github.com/openvinotoolkit/openvino/wiki/CMakeOptionsForCustomCompilation, like: 
 - NGRAPH_ONNX_FRONTEND_ENABLE
 - NGRAPH_PDPD_FRONTEND_ENABLE
 - GRAPH_TF_FRONTEND_ENABLE
 - NGRAPH_IR_FRONTEND_ENABLE
That cmake does not respect anywhere. You can check by downloading the repo and grep -r "<the option>" /.../openvino. You also get the warning from cmake:

~~~bash
cmake -G "Visual Studio 16 2019" -A x64 [...]  -DNGRAPH_ONNX_FRONTEND_ENABLE=OFF -DNGRAPH_PDPD_FRONTEND_ENABLE=OFF -DNGRAPH_TF_FRONTEND_ENABLE=OFF -DCMAKE_BUILD_TYPE=Release ..
~~~

You will  get the warning from cmake:
~~~bash
CMake Warning:
  Manually-specified variables were not used by the project:

    NGRAPH_ONNX_FRONTEND_ENABLE
    NGRAPH_PDPD_FRONTEND_ENABLE
    NGRAPH_TF_FRONTEND_ENABLE
~~~

##### Steps to reproduce
~~~bash
git clone https://github.com/openvinotoolkit/openvino
cd openvino
git checkout tags/2021.4
git submodule update --init --recursive
mkdir build
cd build
cmake using the options above
~~~

##### Issue submission checklist

 - [X] I report the issue, it's not a question
   
 - [X] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   
 - [X] There is reproducer code and related data files: images, videos, models, etc.

**Comment by jgespino** [comment]
Hi @dnns92 

Thanks for reporting, the documentation needs to be updated. Per https://github.com/openvinotoolkit/openvino/pull/9016 the following CMake options were renamed. 
```
NGRAPH_ONNX_FRONTEND_ENABLE -> ENABLE_OV_ONNX_FRONTEND
NGRAPH_PDPD_FRONTEND_ENABLE -> ENABLE_OV_PDPD_FRONTEND
NGRAPH_IR_FRONTEND_ENABLE -> ENABLE_OV_IR_FRONTEND
NGRAPH_TF_FRONTEND_ENABLE -> ENABLE_OV_TF_FRONTEND
NGRAPH_USE_SYSTEM_PROTOBUF -> ENABLE_SYSTEM_PROTOBUF
NGRAPH_UNIT_TEST_ENABLE -> ENABLE_OV_CORE_UNIT_TESTS
NGRAPH_UNIT_TEST_BACKENDS_ENABLE -> ENABLE_OV_CORE_BACKEND_UNIT_TESTS
OPENVINO_DEBUG_ENABLE -> ENABLE_OPENVINO_DEBUG
```

Regards,
Jesus

**Comment by dnns92** [comment]
Dear @jgespino,

thank you for your reply, however: The pull-request you linked has been merged into the main branch december, while the 2021.4-Tag had its latest commit somewhere in June. So that cannot be the reason for the commands missing.

**Comment by jgespino** [comment]
@dnns92 I believe these options were not available in 2021.4 release, if we look at the commit history for this wiki page. The options were added around [November 17th 2021](https://github.com/openvinotoolkit/openvino/wiki/CMakeOptionsForCustomCompilation/6b70a916e50cdd7cc8a2e2c20fa7e2c658917e4b). 


Also, tags 2021.4.1 or 2021.4.2 don't include these options. I don't have the exact pull request where these options were introduced.

**Comment by jgespino** [comment]
@dnns92 The option names have been updated in the wiki page. Please let me know if you have additional questions.
https://github.com/openvinotoolkit/openvino/wiki/CMakeOptionsForCustomCompilation

Regards,
Jesus
